lib/patch-series.ts

```diff
@@ -288,6 +303,12 @@ export class PatchSeries {
                                 }
                             });
                             break;
+                        case "range-diff:":
```

Comment:
Thank you for changing it in the code. Could you also adjust the commit message, please?

Reply:
Doh!